packages/ui-kit/src/hooks/use-iso310-number-formatter.js

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from 'react';
+import { useIntl } from 'react-intl';
+
+// https://en.wikipedia.org/wiki/ISO_31-0#Numbers (now ISO 80000-1, but its text is not public)
+const narrowNonBreakingSpace = '\u202F';
+
+const useISO310NumberFormatter = () => {
```

Comment:
@emmenko  To be able to use the function in node.js preprocessing like a remark plugin, too it would be useful to not only expose the hook but also a plain javascript function that takes a number and returns the string.  What do you think?  

I consider adding a remark plugin that does the number formatting automatically in content.

Reply:
Let's do this as a follow up.